ProjectManagement.BusinessLogic.Tests/Service/BoardServiceTests.cs

```diff
@@ -0,0 +1,546 @@
+﻿using Moq;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using ProjectManagement.BusinessLogic.Services.Implementation;
+using ProjectManagement.DataAccess.Repositories.Interfaces;
+using ProjectManagement.Domain.Models;
+using Xunit;
+using ProjectManagement.BusinessLogic.Services.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using System.Security.Claims;
+using ProjectManagement.BusinessLogic.Specifications;
+using ProjectManagement.BusinessLogic.Exceptions;
+
+namespace ProjectManagement.BusinessLogic.Tests.Service
+{
+    public class BoardServiceTests
+    {
+        private readonly Mock<IUnitOfWork> _unitOfWorkMock;
+        private readonly Mock<IBoardRepository> _boardRepositoryMock;
+        private readonly Mock<IBoardMemberRepository> _boardMemberRepositoryMock;
+        private readonly Mock<IUserRepository> _userRepositoryMock;
+        private readonly IUserManager _userManager;
+        private readonly BoardService _service;
+
+        private int _currentUserId = 1;
+        private string _currentUserName = "Tom";
+
+        public BoardServiceTests()
+        {
+
+            _unitOfWorkMock = new Mock<IUnitOfWork>();
+            _unitOfWorkMock.Setup(x => x.SaveChangesAsync(It.IsAny<CancellationToken>()))
+                .Returns(Task.FromResult(1));
+
+            _boardRepositoryMock = new Mock<IBoardRepository>();
+            _boardRepositoryMock.Setup(x => x.UnitOfWork)
+                .Returns(_unitOfWorkMock.Object);
+
+            _boardMemberRepositoryMock = new Mock<IBoardMemberRepository>();
+            _boardMemberRepositoryMock.Setup(x => x.UnitOfWork)
+                .Returns(_unitOfWorkMock.Object);
+
+            _userRepositoryMock = new Mock<IUserRepository>();
+            _userRepositoryMock.Setup(x => x.UnitOfWork)
+                .Returns(_unitOfWorkMock.Object);
+
+            var mockHttpContextAccessor = new Mock<IHttpContextAccessor>();
+            var claims = new[]
+          {
+                new Claim(ClaimTypes.NameIdentifier, _currentUserId.ToString()),
+                new Claim(ClaimTypes.Name, _currentUserName)
+            };
+            var identity = new ClaimsIdentity(claims, "basic");
+            var principal = new ClaimsPrincipal(identity);
+            var httpContext = new DefaultHttpContext()
+            {
+                User = principal
+            };
+            mockHttpContextAccessor.Setup(_ => _.HttpContext).Returns(httpContext);
+            _userManager = new UserMananger(mockHttpContextAccessor.Object, _userRepositoryMock.Object);
+            _service = new BoardService(_boardRepositoryMock.Object, _boardMemberRepositoryMock.Object, _userManager);
+        }
+
+        [Fact]
+        public async void GetBoardsAsync_Passed()
```

Comment:
Please use some descriptive names for the tests. There is a suggestion from Microsoft docs https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests